TypeCobol.LanguageServer/Workspace.cs

```diff
@@ -250,6 +256,25 @@ public bool TryGetOpenedDocumentContext(Uri fileUri, out DocumentContext openedD
             }
         }
 
+        /// <summary>
+        /// Load Custom Analyzers
+        /// </summary>
+        /// <param name="customAnalyzerFiles"></param>
+        internal void LoadCustomAnalyzers(List<string> customAnalyzerFiles)
+        {
+            if (customAnalyzerFiles != null)
+            {
+                try
+                {
+                    _customAnalyzerProviders = customAnalyzerFiles.Select(f => AnalyzerProviderLoader.LoadProvider(f));
+                }
+                catch(Exception e)
+                {
+                    _Logger(e.Message, null);
+                }
```

Comment:
If we want to log exceptions happening during load,w e have to actually perform the load. LINQ Select is deferred so nothing happens here but on the first iteration over `_customAnalyzerProviders`

Reply:
done in 78ea44d